"make generate" command  run failed

When i run the "make generate" command, it failed because the file of pkg/apis/kubeflow/v2beta1/openapi_generated.go does not exist. 
```shell
$ make generate
go generate ./pkg/... ./cmd/...
hack/update-codegen.sh
~/Desktop/mpi-operator ~/Desktop/mpi-operator
~/Desktop/mpi-operator
Generating deepcopy funcs
Generating apply configuration for kubeflow:v2beta1 at github.com/kubeflow/mpi-operator/pkg/client/applyconfiguration
Generating clientset for kubeflow:v2beta1 at github.com/kubeflow/mpi-operator/pkg/client/clientset
Generating listers for kubeflow:v2beta1 at github.com/kubeflow/mpi-operator/pkg/client/listers
Generating informers for kubeflow:v2beta1 at github.com/kubeflow/mpi-operator/pkg/client/informers
/Library/Developer/CommandLineTools/usr/bin/make manifest
mkdir -p /Users/wenxiaowang/Desktop/mpi-operator/bin
/Users/wenxiaowang/Desktop/mpi-operator/bin/controller-gen "crd:generateEmbeddedObjectMeta=true" paths="./..." output:crd:artifacts:config=manifests/base
hack/generate-manifest.sh /Users/wenxiaowang/Desktop/mpi-operator/bin/kustomize
hack/python-sdk/gen-sdk.sh
Generating V2 OpenAPI specification ...
API rule violation: list_type_missing,github.com/kubeflow/mpi-operator/pkg/apis/kubeflow/v2beta1,TaskProperties,ProcessNumOpt
Generating V2 swagger file ...
# command-line-arguments
hack/python-sdk/main.go:48:23: undefined: mpijobv2.GetOpenAPIDefinitions
make: *** [generate] Error 1
```

--- a/hack/python-sdk/gen-sdk.sh
+++ b/hack/python-sdk/gen-sdk.sh
@@ -29,7 +29,7 @@ if [ -z "${GOPATH:-}" ]; then
 fi
 
 # Backup existing v2 openapi_generated.go
-mv pkg/apis/kubeflow/v2beta1/openapi_generated.go pkg/apis/kubeflow/v2beta1/openapi_generated.go.backup
+cp pkg/apis/kubeflow/v2beta1/openapi_generated.go pkg/apis/kubeflow/v2beta1/openapi_generated.go.backup
 
 CODEGEN_VERSION=$(grep 'k8s.io/code-generator' go.sum | awk '{print $2}' | sed 's/\/go.mod//g' | head -1)
 GOBIN="${PWD}/bin" go install "k8s.io/code-generator/cmd/openapi-gen@${CODEGEN_VERSION}"

